src/Http/Http.Extensions/test/RequestDelegateGenerator/Baselines/Multiple_MapAction_NoParam_StringReturn.generated.txt

```diff
@@ -226,6 +228,7 @@ namespace Microsoft.AspNetCore.Http.Generated
 
                     async Task RequestHandler(HttpContext httpContext)
                     {
+
```

Comment:
Nit: I assume this empty whitespace is from the empty parameter preparation block since this handler takes no parameters.

Reply:
Yep. I think it would make the emission code less readable if we tried to remove this :)